tdclient/connector_api.py

```diff
@@ -78,12 +83,14 @@ def connector_list(self):
             return json.loads(body.decode("utf-8"))
 
     def connector_create(self, name, database, table, job, params=None):
-        """
+        """Create a DataConnector session.
+
         Args:
-          name (str): name of the connector job
-          database (str): name of the database to perform connector job
-          table (str): name of the table to perform connector job
-          job (dict): :class:`dict` representation of `load.yml`
+            name (str): name of the connector job
+            database (str): name of the database to perform connector job
+            table (str): name of the table to perform connector job
+            job (dict): :class:`dict` representation of `load.yml`
+            params (dict, optional): Extra parameters
```

Comment:
Is it possible to list what parameters are available?

Reply:
addedd 5913169